### PR TITLE
Modified changelog workflow

### PR DESCRIPTION
**Pull request by vlebourl**
from https://github.com/47degrees/scalacheck-toolbox/blob/master/.github/workflows/changelog.yml

also added release drafter from https://github.com/47degrees/scalacheck-toolbox/blob/master/.github/workflows/release-drafter.yml

### CODE REVIEW

File: .github/workflows/changelog.yaml [hunk 1]
```diff
@@ -1,38 +1,68 @@
...
+            },
+            "bug": {
+              "prefix": "🐛 **Bug Fixes**",
+              "labels": ["bug"]
+            },
+            "dependency": {
+              "prefix": "📈 **Dependency updates**",
+              "labels": ["dependency-update", "scala-steward"]
+            }
+          }'
```
Comment (tetienne) [inline .github/workflows/changelog.yaml:59]: So we will have to use these labels. We need to create them. We can also added them automatically based on the name of the PR.

Comment (vlebourl) [reply .github/workflows/changelog.yaml:59]: I think we don't need these ones actually, I should remove them. 
We can do the auto labeling based on PR name with this: https://github.com/TimonVS/pr-labeler-action

File: .github/pr-labeler.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+enhancement: ['feature/*', 'feat/*', 'enhancement/*']
```
Comment (tetienne) [inline .github/pr-labeler.yml:1]: I would make an other label for feature. Like this in the changelog you will not see the new feature mixed with the refactors for instance.

File: .github/workflows/changelog.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+              "prefix": "🚀 **Features**",
+              "labels": ["enhancement"]
+            },
+            "documentation": {
+              "prefix": "📘 **Documentation**",
+              "labels": ["documentation"]
+            },
+            "bug": {
+              "prefix": "🐛 **Bug Fixes**",
+              "labels": ["bug"]
```
Comment (tetienne) [inline .github/workflows/changelog.yml:53]: + feature

File: .github/workflows/changelog.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            },
+          }'
+      - name: Create Changelog Pull Request
+        uses: peter-evans/create-pull-request@v2
+        with:
+          token: ${{ secrets.ADMIN_GITHUB_TOKEN }}
+          commit-message: Update CHANGELOG.md [ci skip]
```
Comment (tetienne) [inline .github/workflows/changelog.yml:60]: I think you can removed the [ci skip]

Comment (vlebourl) [reply .github/workflows/changelog.yml]: done

File: .github/workflows/changelog.yml (hunk 3 above)
Comment (vlebourl) [reply .github/workflows/changelog.yml:57]: done

File: .github/pr-labeler.yml (hunk 2 above)
Comment (vlebourl) [reply .github/pr-labeler.yml]: done

File: .github/workflows/changelog.yml [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+            },
+            "bug": {
+              "prefix": "🐛 **Bug Fixes**",
+              "labels": ["bug"]
+            },
+          }'
+      - name: Create Changelog Pull Request
+        uses: peter-evans/create-pull-request@v2
+        with:
+          token: ${{ secrets.ADMIN_GITHUB_TOKEN }}
```
Comment (tetienne) [inline .github/workflows/changelog.yml:63]: @iMicknl Do we have this token?

Comment (iMicknl) [reply .github/workflows/changelog.yml:63]: No, what is needed here?

Comment (vlebourl) [reply .github/workflows/changelog.yml:63]: from this https://github.com/peter-evans/create-pull-request I think we can use the defaut GITHUB_TOKEN, right?

Comment (tetienne) [reply .github/workflows/changelog.yml:63]: I think so

**Pull request merged**